src/Sheet/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { FocusOn } from 'react-focus-on';
+
+import SheetContainer from './SheetContainer';
+
+export const POSITIONS = {
+  left: 'left',
+  right: 'right',
+  top: 'top',
+  bottom: 'bottom',
+};
+
+export const VARIANTS = {
+  light: 'light',
+  dark: 'dark',
+};
+
+class Sheet extends React.Component {
+  constructor(props) {
+    super(props);
+    this.renderSheet = this.renderSheet.bind(this);
+  }
+
+  renderSheet() {
+    const { children, position, variant } = this.props;
+    return (
+      <div
+        className={classNames(
+          'pgn__sheet-component',
+          `pgn__sheet__${variant}`,
+          position,
+        )}
+        role="alert"
+        {...{
+          'aria-live': 'polite',
+          'aria-atomic': 'true',
+        }}
```

Comment:
```suggestion
        aria-live="polite"
        aria-atomic="true"
```
You shouldn't need to do this, the linter should accept aria attributes on jsx